VMBackup/main/Utils/SizeCalculation.py

```diff
@@ -46,10 +47,35 @@ def __init__(self,patching, hutil, logger,para_parser):
             errMsg = 'Failed to serialize customSettings with error: %s, stack trace: %s' % (str(e), traceback.format_exc())
             self.logger.log(errMsg, True, 'Error')
             self.isOnlyOSDiskBackupEnabled = False
-
+        # The command lsscsi is used for mapping the LUN numbers to the disk_names
+        self.command = "sudo lsscsi"
+        self.disksToBeIncluded = []
+        self.root_devices = []
+        self.root_mount_points = ['/' , '/boot/efi']
+        try:
+            self.output_lsscsi = (os.popen(self.command).read())
+            self.lsscsi_list = self.output_lsscsi.splitlines()
+        except Exception as e:
+            error_msg = "Failed to execute the command lsscsi because of error %s , stack trace: %s" % (str(e), traceback.format_exc())
+            self.logger.log(error_msg, True ,'Error')
+            self.output_lsscsi = ""
+            self.lsscsi_list = []
+        try:
+             self.output_lsblk = json.loads(os.popen("lsblk --json").read())
```

Comment:
[SUGGESTION] - Since all thats used from the output of this command below ( lines 159-169) are the keys "name" and "mountpoint" - maybe use `lsblk --json --output name,mountpoint` instead. Should reduce the memory impact of storing the output of this by a few times. Include more keys when needed.